Fix  #1092 Update Python/Django versions in tests

pauloxnet opened the pull request.

jschneier commented on setup.cfg:
```diff
@@ -30,7 +28,7 @@ classifiers =
-python_requires = >=3.5
+python_requires = >=3.6
 install_requires =
     Django >= 2.2
```
We should drop 3.0 here

pauloxnet replied:
I updated the PR

browniebroke commented:
Is there anything left to do? How can we help to get this merged?

jschneier commented:
There are a couple things I think we should update before merging.

1. `install_requires` does not properly drop Django versions
2. CHANGELOG updates should not include `1.12.4`, I wouldn't release a Django or Python dropping version as a patch (did that once and was called out for it, rightly)

pauloxnet commented:
> There are a couple things I think we should update before merging.

Thanks for the feedback, I updated this PR.

> 1. `install_requires` does not properly drop Django versions

I updated the Django versions in install_requires

> 2. CHANGELOG updates should not include `1.12.4`, I wouldn't release a Django or Python dropping version as a patch (did that once and was called out for it, rightly)

I updated to version 1.13

I also rebased the branch to let you merge it.

jschneier commented:
Sorry, one more thing. Can you please revert the change to `__version__` and setting the version in `CHANELOG.rst`. For the second you can just put `XXXX-XX-XX`.

pauloxnet commented:
> Can you please revert the change to `__version__` and setting the version in `CHANELOG.rst`. For the second you can just put `XXXX-XX-XX`.

I reverted the changes on the version, I rebased to the branch master to resolve a merge conflict and I also added the support to Django 4.1 which was published yesterday.

jschneier commented:
@pauloxnet thanks for the persistence. There is now a test failure.

pauloxnet commented:
> @pauloxnet thanks for the persistence. There is now a test failure.

@jschneier Actually I had forgotten how old this PR is. I realized that in the meantime Django 2.2 is no longer supported, so I updated this PR. Locally the tests pass, I hope they pass even remotely. Unfortunately, as first-time Contributor for this project they will not automatically start, so I will have to wait for your approval for be sure.
Anyway thanks for your patience.

jschneier commented:
Merged, thanks for the patience.

pauloxnet commented:
Thanks to you
I am happy to have been able to contribute
